### DISCUSSION TITLE
PVTable modify

### DISCUSSION THREAD

**Discussion by joshikk** [ControlSystemStudio/phoebus#2123] [Q&A]
Hello,
I am able to get the PVTable in the table widget wuing pvaccess protocol,
I am wondering in what way can I modify the cells/table so that it can get updated on the PVAServer

Thanks in advance.

PS: With EPICS 7, it will be great if phoebus has additional API's for interfacing complex structures of PVAccess. It will help if anyone can point to the proper link/demo code

**Comment by kasemir** [comment]
There is at this time no support for writing NTTable data types, see #1214
